python/tvm/relay/frontend/pytorch.py

```diff
@@ -105,6 +105,36 @@ def _impl(inputs, input_types):
         return _op.transform.strided_slice(data, begin, end, strides)
     return _impl
 
+def _split():
+    def _impl(inputs, input_types):
+        data = inputs[0]
+        split_size = int(inputs[1])
+        dim = int(inputs[2])
+
+        now_indice = split_size
+        indices = []
+        while now_indice < _infer_shape(data)[dim]:
+            indices.append(now_indice)
+            now_indice += split_size
+
+        return _op.split(data, indices, dim)
```

Comment:
Because this op is splitting the index evenly, you can just provide a number of sections to split to rather than a list of indices.
```
return _op.split(data, int(_infer_shape(data)[dim] / split_size), dim)
```

Reply:
The split op in Relay and PyTorch have different behaviors.  In Relay, if indices_or_sections is an integer, the input will be divided equally along the given axis.  But in Pytorch, if split_size_or_sections is an integer type, then tensor will be split into equally sized chunks (if possible). The last chunk will be smaller if the tensor size along the given dimension dim is not divisible by split_size.